.github/workflows/docker.yaml

```diff
@@ -1,6 +1,5 @@
 name: Docker
 on:
-  pull_request:
```

Comment:
What is the reason for removing this?

Reply:
I don't feel super strongly about this change. It was to reduce the overall amount of action runtime. This build is never pushed anywhere, so if we wanted to build and test an image there's a separate `docker_test` wf that will post to `images-dev`. All this one is really doing is proving that a pip install works, which doesn't feel worthwhile to do on each commit.

Reply:
Works for me. I just like to have the reasons for things spelt out (ideally in the commit messages), for posterity.
(i.e., for future me trying to figure out what's happened…)

Reply:
aye, that's fair. I was going to add something informative for the removal, but i couldn't think of why this had ever been there in the first place 😓